specification/loadtestservice/LoadTestService.Management/main.tsp

```diff
@@ -0,0 +1,31 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./routes.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+
+@armProviderNamespace
+@service({
+  title: "LoadTest Client",
```

Comment:
Nit, please remove the word client from here

Reply:
"LoadTest Client" is already our title in published spec. Should I update this?

Reply:
If you're planning on generating client libraries, the emitters use the title to create the client name so you'd have to test it out but having `Client` at the end might cause conflicts

Reply:
we already have sdks published for GA spec